parm/config/gefs/yaml/defaults.yaml

```diff
@@ -15,4 +15,4 @@ base:
   REPLAY_ICS: "NO"
   USE_OCN_PERTURB_FILES: "false"
   HPSSARCH: "NO"
-  LOCALARCH: "NO"
+  LOCALARCH: "YES"
```

Comment:
This is fine for testing, but we will want `HPSSARCH` on by default, not `LOCALARCH`.